src/freemindlatex/__main__.py

```diff
@@ -110,9 +160,22 @@ def _CompileInWorkingDirectory(work_dir):
         "mindmap.mm"),
       'r',
       'utf8').read())
-  org.OutputToBeamerLatex(os.path.join(work_dir, "mindmap.tex"))
+  output_tex_file_loc = os.path.join(work_dir, "mindmap.tex")
+  org.OutputToBeamerLatex(output_tex_file_loc)
+
+  try:
+    _CompileLatexAtDir(work_dir, "slides.tex")
+  except LatexCompilationError as e:
```

Comment:
This section is a little too involved. Pull the retry logic into a different function.